apps/nameguard.io/src/components/organisms/ContactUsForm.tsx

```diff
@@ -274,30 +253,19 @@ export const ContactUsForm = ({ title }: ContactUsFormProps) => {
               >
                 Telegram (optional)
               </label>
-              <div className="relative">
-                <span className="text-sm leading-5 font-medium text-gray-500 absolute left-2 top-2.5">
-                  @
-                </span>
-                <input
-                  type="text"
-                  id="telegram"
-                  autoComplete="off"
-                  disabled={isLoading}
-                  onChange={handleInputChange}
-                  name={FormFields.Telegram}
-                  className={cc([
-                    "block w-full rounded-md pr-3.5 pl-6 py-2 text-gray-900 shadow-sm border border-gray-300 placeholder:text-gray-400 focus:border-gray-600 hover:border-gray-400 outline-none sm:text-sm sm:leading-6",
-                    {
-                      "border-red-300": validationErrors[FormFields.Telegram],
-                    },
-                  ])}
-                />
-                {validationErrors[FormFields.Telegram] && (
-                  <span className="mt-2 text-sm font-normal text-red-600">
-                    {validationErrors[FormFields.Telegram]}
-                  </span>
-                )}
-              </div>
+              <Input
+                type="text"
+                id="telegram"
+                slotPosition="left"
+                slot="@"
+                autoComplete="off"
+                disabled={isLoading}
```

Comment:
The disabled state when loading isn't working for me in the preview environment.

It might be that we need to add support for this case and define the styling for it inside the <Input> component.

Appreciate your advice and suggestions. Thank you :+1:

Reply:
@lightwalker-eth I created a disabled version, both for `Input` and `TextArea`. Let me know what you think :)